app/metrics.yaml

```diff
@@ -7278,3 +7278,31 @@ wallpapers:
     data_sensitivity:
       - interaction
     expires: "2023-05-01"
+
+recently_visited_homepage:
```

Comment:
nit: does this need to end with "_homepage:"?  Looking at other examples like "recent_searches:" don't specify home.  Also should we move this block near the category "recent_searches:" since we have "recent_bookmarks", "recent_tabs" around there.

Reply:
There was already one recently_visited

Reply:
I see, that's unfortunate.